Resources/Locale/en-US/identity/identity-block.ftl

```diff
@@ -0,0 +1,6 @@
+identity-block-examinable-verb-text = Concealment
+
+identity-block-coverage-text-none = This item does not cover up anything.
+identity-block-coverage-text-mouth = This item covers the mouth.
+identity-block-coverage-text-eyes = This item covers the eyes.
+identity-block-coverage-text-full = This item covers the entire face.
```

Comment:
```suggestion
identity-block-coverage-text-mouth = This item hides the lower half of the face.
identity-block-coverage-text-eyes = This item hides the upper half of the face.
identity-block-coverage-text-full = This item hides the entire face.
```

It's a minor thing but I think splitting it into lower/upper face makes it clearer that you need two pieces, and "hides" is more closely related to concealment.